client/js/board.js

```diff
@@ -68,18 +69,19 @@ export default class Board {
 	}
 
 	createScoreboard() {
-        this.scoreboard = new Scoreboard();
+		this.scoreboard = new Scoreboard();
 
-        this.scoreboard.playersContainer.appendTo('#scoreboard');
-    }
+		this.scoreboard.playersContainer.appendTo('#scoreboard');
+	}
 
 	render() {
-
 		setInterval(() => {
-            // TEMP: ONLY START MOVING THE FIRST SNAKE FOR TEST PURPOSES
-			this.snakes[0].move(this);
-			this.scoreboard.updateScores(this.snakes);
-			this.checkSnakeSelfCollision();
+			if(this.shouldRender) {
```

Comment:
setInterval return an interval identifier. It can be stopped by calling `clearInterval`.
See https://developer.mozilla.org/en-US/docs/Web/API/WindowTimers/clearInterval